docs/packaging/package.yml.md

```diff
@@ -85,34 +86,34 @@ The packaging steps are all considered optional, however the absence of the `ins
 | Step Name   | Description                                                                                                                                                |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **setup**   | Performed after the source extraction. This is the correct place to perform any `configure` routine, or to `patch` the sources.                            |
-| **build**   | Use this step to run the build portion, i.e. `make`                                                                                                        |
-| **install** | This is where you should install the files into the final packaging directory, i.e. `make install`                                                         |
-| **check**   | This is where tests / checking should occur, i.e. `make check`                                                                                             |
+| **build**   | Use this step to run the build portion, e.g. `make`                                                                                                        |
```

Comment:
Let's avoid Latin. Please replace "e.g". with "for example", surrounded by commas, throughout this page.